### PR TITLE
prov/verbs: Add invocation of ibv_fork_init() before other libibverbs functions. Minor enhancement in the logging

### PR DESCRIPTION
**Pull request by gladkovdmitry17**

### CODE REVIEW

File: prov/verbs/src/verbs_cm.c [hunk 1]
```diff
@@ -244,28 +244,32 @@ static int fi_ibv_pep_setname(fid_t pep_fid, void *addr, size_t addrlen)
...
 		ret = rdma_destroy_id(pep->id);
 		if (ret) {
-			FI_INFO(&fi_ibv_prov, FI_LOG_FABRIC, "Unable to destroy previous rdma_cm_id\n");
+			VERBS_INFO(FI_LOG_FABRIC,
+				   "Unable to destroy previous rdma_cm_id\n");
 			return -errno;
 		}
-		ret = rdma_create_id(NULL, &pep->id, &pep->pep_fid.fid, RDMA_PS_TCP);
+		ret = rdma_create_id(NULL, &pep->id,
+				     &pep->pep_fid.fid, RDMA_PS_TCP);
```
Comment (a-ilango) [inline prov/verbs/src/verbs_cm.c]: Indent not needed. The line fits within 80 columns.

**Comment by a-ilango** [comment]
Except for one minor comment, rest of the code looks fine. Also this PR has some conflicts with the master.

File: prov/verbs/src/verbs_cm.c (hunk 1 above)
Comment (gladkovdmitry17) [inline prov/verbs/src/verbs_cm.c]: Ok, I'll return back prior indent

**Comment by gladkovdmitry17** [comment]
Thank you, @a-ilango . Please, find PR updated as requested

**Comment by shefty** [comment]
Thanks Dmitry!

**Pull request merged**